force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -16031,7 +16031,7 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Questions Answered link label in Get Started Resources section for End Users</shortDescription>
-        <value>Trailblazer Community</value>
```

Comment:
Is there a reason this specific label is switching back to Trailblazer Community? I see the link corresponding to this label is still redirected to the Trailblazer Community page.

Reply:
Great question, I took this from a Quip Doc that was provided, I just asked for confirmation of the change there.

Reply:
yeah it was my understanding that we've now changed all URLs to point to Trailblazer Community and not Power of Us Hub

Reply:
I haven't heard back about this, but I also think we should keep it as Trailblazer Community, this was probably a mistake in the Doc that I was working from. I'll make that revert tomorrow!